### ISSUE TITLE
This code is not working anymore :(

### ISSUE THREAD

**Issue by AviSub** [almog/yad2.rss#6]
When trying to install the bundle it keep saying that there are some missing old gems
anyone saw this before?

**Comment by almog** [comment]
Yes, I'm aware of that, even if dependencies were in place, yad2 has gone through an overhaul. 
I have plans to rewrite it but in the meanwhile it's broken.

**Comment by AviSub** [comment]
Can I help somehow?


Thanks ,
Avi Sabaj

Escalation Engineering
DellEMC | RecoverPoint
Office: +972-8-6242891
Mobile: +972-54-7559504
________________________________

**Comment by almog** [comment]
I think I'll start from scratch, but I I'm not sure when I'll begin doing so, probably no later than when I start looking for an apartment. :)

If you do want to use the existing code with minimal effort to see if it's still viable, you could probably fix the dependencies issues easily, and try to expose just a single field to see how things have changes on yad2's end.
